source/advanced-topics.adoc

```diff
@@ -1143,3 +1143,77 @@ The `%ifos` conditional is used to control processing based on the operating sys
   %endif
 
 All the contents of the SPEC file between `%ifos` and `%endif` are processed only if the build was done on a Linux system.
+
+[id="Packaging-of-Python-RPMs"]
+=== Packaging of Python RPMs
+
+[id="Benefits-of-packaging-Python-packages-as-RPMs"]
+==== Benefits of packaging Python packages as RPMs
+
+Benefits of packaging Python packages as RPMs are the same as for any other RPM package. More can be found in the chapter xref:Why-Package-Software-with-RPM[].
+
+[id="Converting-PyPI-package-to-an-RPM"]
+==== Converting PyPI package to an RPM
+
+Pyp2rpm is a tool which automatically converts PyPI packages to an RPM spec file. Simple tutorial can be found on this page link:https://github.com/fedora-python/pyp2rpm[pyp2rpm].
```

Comment:
I'd just add a note that some manual corrections may be necessary, to not set too high expectations.

Reply:
"Pyp2rpm is still in development. Some manual corrections might be necessary."